prometheus/base/alerting/prometheus.argoCD_alerts.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    openshift.io/prometheus-rule-evaluation-scope: leaf-prometheus
+  name: o11y-argoCD-alerting-rules
+  namespace: o11y
+spec:
+  groups:
+  - name: argoCD_alerts
+    rules:
+    - alert: DegradedArgoCDApp
+      expr: max_over_time(argocd_app_info{health_status="Degraded", dest_namespace!~".+-tenant"}[5m]) == 1
+      for: 15m
+      labels:
+        severity: Critical
+      annotations:
+        summary: "ArgoCD service at {{ $labels.dest_namespace }} namespace includes an app which was 'Degraded'"
+        description: "ArgoCD service at {{ $labels.dest_namespace }} namespace includes an app which was 'Degraded' in the last 5 minutes"
```

Comment:
You can get the name of the application from the metric and produce more precise summary and descriptions.  
Maybe something like:
`description: "Application {{ $labels.name }} in namespace {{ $labels.dest_namespace }} was 'Degraded' in the last 5 minutes"`